src/runtime_src/xrt/device/hal2.cpp

```diff
@@ -139,7 +139,7 @@ void
 device::
 release_cu_context(const uuid& uuid,size_t cuidx)
 {
-#if 0
+#if 1
```

Comment:
Enabling this implies we are confident mailbox works.

Reply:
Yes it does, but we are still protected by XCL_MULTIPROCESS_MODE which we should remove once MailBox is solid.